modules/apps/portal-store/portal-store-azure/bnd.bnd

```diff
@@ -45,55 +35,29 @@ Import-Package:\
 	\
 	!kotlin.collections,\
 	\
-	!kotlin.jvm,\
-	!kotlin.jvm.functions,\
-	!kotlin.jvm.internal,\
-	!kotlin.jvm.internal.markers,\
+	!kotlin.jvm.*,\
 	\
 	!kotlin.reflect,\
 	\
 	!kotlin.sequences,\
 	\
-	!lzma.sdk,\
-	!lzma.sdk.lzma,\
+	!lzma.sdk.*,\
 	\
-	!net.jpountz.lz4,\
-	!net.jpountz.xxhash,\
+	!net.jpountz.*,\
 	\
 	!net.shibboleth.utilities.java.support.xml,\
 	\
-	!org.apache.commons.compress.compressors,\
-	!org.apache.commons.compress.utils,\
-	!org.apache.commons.csv,\
+	!org.apache.commons.*,\
```

Comment:
@javalosjr96, Change the below packages for `!org.apache.commons.*,\`. It seems be not good, since `org.apache.commons.logging` is imported to generated Import-Package from MANIFEST.MF.

```
!org.apache.commons.compress.compressors,\
	!org.apache.commons.compress.utils,\
	!org.apache.commons.csv,\
```
I don't t understand the reason since we have defined  `!org.apache.commons.*,\` in bnd.bnd. 

I added them back and removed them. Please see https://github.com/liferay-core-infra/liferay-portal/pull/3688